Singlelinkedlist.h

```diff
@@ -0,0 +1,28 @@
+
+#ifndef SINGLELINKEDLIST_H_
+#define SINGLELINKEDLIST_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct Node
```

Comment:
```suggestion
typedef struct NodeS
```
Dodaj S na końcu nazw struktury